prover_rust/Makefile

```diff
@@ -0,0 +1,43 @@
+.PHONY: prover
+
+ifeq (4.3,$(firstword $(sort $(MAKE_VERSION) 4.3)))
+    $(info ************  First ************)
+    HALO2_VERSION=$(shell grep -m 1 "halo2.git" ./Cargo.lock | cut -d "#" -f2 | cut -c-7)
+else
+    $(info ************  Second ************)
```

Comment:
forget to clean up?>

Reply:
Yes this is a debug info, however I think it's ok to keep this line. I was a little confused not knowing which branch the  make use...

Reply:
but now you already know why we add this "if branch"
I suggest we remove this debug info, or at least make the message more meaningful
otherwise it can be confusing for other readers

Reply:
Make sense, I will delete it then.